platform/core/src/core.rs

```diff
@@ -17,14 +39,35 @@ fn get_instance() -> Result<&'static Core, CoreFailure> {
     }
 }
 
-pub fn bootstrap(language_code: String) -> Result<(), CoreFailure> {
+pub fn bootstrap(
+    language_code: String,
+    platform_hook: Box<dyn PlatformHookRequest>,
+) -> Result<(), CoreFailure> {
     if CORE.get().is_some() {
         return Err(CoreFailure::core_already_bootstrapped());
     }
 
-    let core = Core { language_code };
+    #[cfg(target_os = "android")]
+    android_logger::init_once(Config::default().with_min_level(Level::Trace));
+    #[cfg(target_os = "android")]
+    trace!("Rust:core => Bootstrapped!");
```

Comment:
Maybe here it can also be like:
```rust
#[cfg(target_os = "android")]
{
     android_logger::init_once(Config::default().with_min_level(Level::Trace));
     trace!("Rust:core => Bootstrapped!");
}
```